input/system/heroku/logs.go

```diff
@@ -173,7 +173,7 @@ func logStreamItemToLogLine(item HerokuLogStreamItem, servers []*state.Server, s
 	}
 	backendPid, _ := strconv.ParseInt(parts[1], 10, 32)
 
-	contentParts := regexp.MustCompile(`^\[(\w+)\] \[(\d+)-(\d+)\] ( sql_error_code = \w+ (\w+):  )?(.+)`).FindStringSubmatch(string(item.Content))
+	contentParts := regexp.MustCompile(`^\[(\w+)\] \[(\d+)-(\d+)\] (?:( sql_error_code = \w+ (\w+):  )|( sql_error_code = \w+ time_ms = ".+?" pid=".+?" proc_start_time=".+?" session_id=".+?" vtid=".+?" tid=".+?" log_line=".+?" .+?database=".+?" connection_source=".+?" user=".+?" application_name=".+?" ))?(.+)`).FindStringSubmatch(string(item.Content))
```

Comment:
Not sure about this, since it seems like an ad-hoc copy of the regex in `parse.go`. Testing it now with a Heroku DB. Unfortunately the Saascontrol primary has already been updated, so I'm not sure if we have an old Heroku database we can test with to ensure this does not break the old prefix handling.